elements/src/components/pos-login-form/pos-login-form.tsx

```diff
@@ -0,0 +1,49 @@
+import { Component, Event, EventEmitter, h, Prop, State, Watch } from '@stencil/core';
+
+@Component({
+  tag: 'pos-login-form',
+  shadow: {
+    delegatesFocus: true,
+  },
+})
+export class PosLoginForm {
+  @State() idpUrl: string = "http://localhost:3000";
+
+  @State() canSubmit: boolean = true;
+
+  /**
+   * Emits the selected IDP URL to use for login
+   */
+  @Event({ eventName: 'idpUrlSelected' }) idpUrlSelected: EventEmitter;
+  @Event({ eventName: 'pod-os:error' }) errorEmitter: EventEmitter;
+
+  @Watch('idpUrl')
+  validate() {
+    this.canSubmit = Boolean(this.idpUrl);
+  }
+
+  render(): any {
+    return (
+      <form method="dialog" onSubmit={e => this.handleSubmit(e)}>
+        <label htmlFor="idpUrl">URL</label>
+        <input id="idpUrl" type="text" value={this.idpUrl} onInput={e => this.handleChange(e)} />
+        <input id="login" type="submit" value="Login" disabled={!this.canSubmit} />
+      </form>
+    );
+  }
+
+  handleChange(event) {
+    this.validate()
+    this.idpUrl = event.target.value;
+  }
+
+  async handleSubmit(event) {
+    try {
+      this.idpUrlSelected.emit(this.idpUrl);
+    } catch (error) {
```

Comment:
I think we do not need the error handling here, since we are only emitting an event.

Reply:
Yeah, I think I got a bit confused while copying pos-new-thing-form